src/Microsoft.Health.Dicom.Core/Features/Common/IWorkItemStore.cs

```diff
@@ -15,6 +15,23 @@ namespace Microsoft.Health.Dicom.Core.Features.Common
     /// </summary>
     public interface IWorkitemStore
```

Comment:
Should this be in WorkItem folder? It does not feel common

Reply:
Yes, it is moved